packages/visitors/visitors.20170828/descr

```diff
@@ -0,0 +1,2 @@
+An OCaml syntax extension (technically, a ppx_deriving plugin) which generates
```

Comment:
The first line of the `descr` field is actually for the short description shown when you do `opam list`. Would it be possible to add that, or to split the above sentence into two lines so that the formatting is correct? Thanks!